### PR TITLE
K-Means binding

### PR DESCRIPTION
**Pull request by Atanahel**
Did the binding for the beautiful K-means implementation of VLFeat (needed it for large scale SIFT clustering).

First time I worked with Cython but was a great way to learn. Mostly copied the way you did things for hog and sift. Not too sure about my test cases.

Comments more than welcomed :-)

### CODE REVIEW

File: cyvlfeat/kmeans/__init__.py [hunk 1]
```diff
@@ -1 +1 @@
-__author__ = 'pts08'
+from .kmeans import kmeans
```
Comment (patricksnape) [inline cyvlfeat/kmeans/__init__.py]: Missing a new line

**Comment by patricksnape** [comment]
Well other than that one missing new line this looks really awesome! Thanks a lot! I assume it is working for you and you are getting results consistent with what you find in Matlab?

Once Windows passes I will probably just merge this in. Thanks again!

**Comment by Atanahel** [comment]
I don't use MATLAB anymore (which is the reason I'm interested in this port). And haven't used this implementation yet in production (though I'm quite confident it works properly by now).

I use some of your conda packages very often (especially the OpenCV3 one) so I feel it's a small payback ;-)

Thanks for the feedback

**Comment by patricksnape** [comment]
Cancelled one just to speed up appveyor - it didn't actually fail.

I mean your test seems pretty solid to me so I think I will just merge when this completes 👍

**Pull request merged**